Mathlib/Tactic/FunProp/Types.lean

```diff
@@ -19,14 +19,11 @@ open Lean Meta
 namespace Meta.FunProp
 
 
-initialize registerTraceClass `Meta.Tactic.fun_prop.attr
 initialize registerTraceClass `Meta.Tactic.fun_prop
-initialize registerTraceClass `Meta.Tactic.fun_prop.step
-initialize registerTraceClass `Meta.Tactic.fun_prop.unify
-initialize registerTraceClass `Meta.Tactic.fun_prop.discharge
-initialize registerTraceClass `Meta.Tactic.fun_prop.apply
-initialize registerTraceClass `Meta.Tactic.fun_prop.unfold
+initialize registerTraceClass `Meta.Tactic.fun_prop.attr
 initialize registerTraceClass `Meta.Tactic.fun_prop.cache
+initialize registerTraceClass `Meta.Tactic.fun_prop.errors
+initialize registerTraceClass `Meta.Tactic.fun_prop.step
```

Comment:
1. Should these have `(inherited := true)` so that `set_option trace.Meta.Tactic.fun_prop` enables all of them? (see [docstring of `registerTraceClass`](https://leanprover-community.github.io/mathlib4_docs/Lean/Util/Trace.html#Lean.registerTraceClass))
2. These could benefit from docstrings explaining what sort of traces they enable.

Reply:
What is the intended way to add doc to an option? 

Like this?
```
/-- bla bla bla -/
initialize registerTraceClass `Meta.Tactic.fun_prop
```

Also I will just cut the tracing options to
```
Meta.Tactic.fun_prop
Meta.Tactic.fun_prop.attr
Debug.Meta.Tactic.fun_prop
```
I don't think there is a need for more right now.

Reply:
Thanks that makes it much more understandable!

Yes, if you add a docstring `/-- yada yada -/` to `initialize registerTraceClass` I think it appears on hover. However, this is completely optional for this PR!